public/pl/_menu.hbs

```diff
@@ -2,6 +2,7 @@
 <a href="/pl/" class="top-nav-item dropdown-toggle" id="nav-gdb" data-toggle="dropdown" data-flip="false" aria-haspopup="true" aria-expanded="false">Biblia Dysforii Płciowej</a>
 <div class="dropdown-menu" aria-labelledby="nav-gdb">
   <a href="/pl/do-druku"                 class="{{#is meta.url '/pl/do-druku'                 }}active {{/is}}dropdown-item">Zobacz wszystko</a>
+  <a href="/gdb-pl.pdf"                     class="dropdown-item">Ściągnij PDF {{icon 'pdf' size="1em" style="vertical-align:middle;margin-bottom:3px;"}} </a>

```

Comment:
"Ściągnij PDF" is perfectly fine, but "Pobierz PDF" would sound even more natural.

Reply:
Thanks! I changed the label.